src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (num1) => {
```

Comment:
```suggestion
  return (firstNumber) => {
```
use more descriptive names